analyzer/src/main/kotlin/managers/Maven.kt

```diff
@@ -171,7 +172,20 @@ class Maven(
             scopes = scopesByName.values.toSortedSet()
         )
 
-        return listOf(ProjectAnalyzerResult(project, packagesById.values.toSortedSet()))
+        val packages = packagesById.values.toSortedSet()
+        val issues = packages.mapNotNull { pkg ->
+            if (pkg.description == "POM was created by Sonatype Nexus") {
+                createAndLogIssue(
+                    managerName,
+                    "Package '${pkg.id.toCoordinates()}' seem to use an auto-generated POM which might lack metadata.",
+                    Severity.WARNING
```

Comment:
Maybe just HINT instead of WARNING because it's just indication of potential other issues?

Reply:
I'm fine with `HINT`, done.